pkgs/applications/audio/soundtracker/default.nix

```diff
@@ -8,19 +8,21 @@
 , jack2
 , audiofile
 , goocanvas # graphical envelope editing
+, libxml2
+, libsndfile
 }:
 
 stdenv.mkDerivation rec {
   pname = "soundtracker";
-  version = "1.0.3";
+  version = "1.0.4";
 
-  src = fetchurl {
+  src = fetchzip {
```

Comment:
may I ask why change to fetchzip ?

Reply:
It makes debugging easier: I can just build soundtracker.src and inspect the source without unpacking it somewhere else first.

It also avoids problems with unstable generated tarballs and should result in better compression (our cache is compressed with zstd iirc). Not the case with this package.

As far as I know there are no disadvantages.

Reply:
ok thank you for clarification, i think for consumers(users) there is no difference